wandb/data_types.py

```diff
@@ -2143,6 +2152,54 @@ def to_json(self, run):
         return json_dict
 
 
+class Bokeh(Media):
+    """
+    Wandb class for Bokeh plots.
+
+    Arguments:
+        val: Bokeh plot
+    """
+
+    artifact_type = "bokeh-file"
+
+    def __init__(self, val):
+        super(Bokeh, self).__init__()
+        bokeh = util.get_module("bokeh")
```

Comment:
We should add the "required" parameter here so we'll throw an exception if bokeh isn't installed.

Reply:
Oh, good call - i added in next commit.